### PR TITLE
Added plugin doc for supervisor flavor

### PR DESCRIPTION
**Pull request by lintongj**
Added plugin doc for supervisor flavor

Signed-off-by: Lintong Jiang <lintongj@vmware.com>

### CODE REVIEW

File: docs/supervisor.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+3. [Install](#install)
+4. [Uninstall](#uninstall)
+5. [Backup](#backup)
+6. [Restore](#restore)
+
+## Compatibility
+
+| vSphere Version |                           vSphere CSI Version                          | Kubernetes Version | Velero Version | Velero Plugin for vSphere Version |
+|:---------------:|:----------------------------------------------------------------------:|:------------------:|:--------------:|:---------------------------------:|
+|  vSphere 7.0 U1c/P02 + ESXi 7.0 U1c/P02 | Bundled with vSphere |     Bundled with vSphere    |     v1.5.1     |         v1.1.0 and higher         |
```
Comment (xing-yang) [inline docs/supervisor.md:15]: v1.5.1 -> v1.5.1 and higher

Comment (xing-yang) [reply docs/supervisor.md:15]: Bundled with vSphere  -> Bundled with vSphere  (v1.16-v1.19)

File: docs/change-block-tracking.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Velero vSphere Change Block Tracking
```
Comment (xing-yang) [inline docs/change-block-tracking.md:1]: Maybe we should remove this section?

File: docs/supervisor.md (hunk 1 above)
Comment (lintongj) [reply docs/supervisor.md]: > v1.5.1 -> v1.5.1 and higher

Leave it as it is based on offline discussion.

File: docs/velero-vsphere-operator-cli.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+
+Below is an example,
+```
+velero-vsphere uninstall -n velero
+```
+**Note**: users are expected to delete the corresponding Supervisor namespace via vSphere UI/API/DCLI after
+running the command above to uninstall Velero.
+
+## Configure Velero & Plugins
+Currently, only toggling CBT configuration in guest clusters (vSphere specific) are supported in the **configure** command.
```
Comment (xing-yang) [inline docs/velero-vsphere-operator-cli.md:147]: Remove any mentioning of CBT/cbt to avoid confusion.

**Comment by svcbot-qecnsdp** [comment]
Congratulations !! 

All the pre-check tests passed
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/432/

**Review by deepakkinni — APPROVED** [review]
apart from Xing's comments, LGTM.

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/434/

**Comment by svcbot-qecnsdp** [comment]
Congratulations !! 

All the pre-check tests passed
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/437/

**Pull request merged**